src/pages/our_ambassadors.astro

```diff
@@ -1610,6 +1610,20 @@ import AmbassadorCard from "@components/AmbassadorCard/index.tsx";
         is a molecular biologist and bioinformatician using nextflow for projects focusing on biodiversity conservation
         and equity in health-care.
       </AmbassadorCard>
+
+      <AmbassadorCard
+        client:load
+        name="Mahima Sanjay Gomladu"
+        img="mahima.png"
+        country="in"
```

Comment:
This suggestion updates your profile flag to the Norwegian flag, which is your country of residence.
```suggestion
        country="no"
```